### PR TITLE
[[AR]] - Spring boot changes alone

### PR DESCRIPTION
**Pull request by aravindan89**
### Change description ###
Spring boot changes alone

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by AyanDuttaHMCTS — APPROVED** [review]

File: api/src/main/java/uk/gov/hmcts/payment/api/dto/mapper/CreditAccountDtoMapper.java [hunk 1]
```diff
@@ -216,12 +216,12 @@ public FeeDto toFeeDto(PaymentFee fee) {
     @SneakyThrows(NoSuchMethodException.class)
     private PaymentDto.LinkDto cancellationLink(String userId, Integer paymentId) {
         Method method = CreditAccountPaymentController.class.getMethod("cancel", String.class, Integer.class);
-        return new PaymentDto.LinkDto(ControllerLinkBuilder.linkTo(method, userId, paymentId).toString(), "POST");
+        return new PaymentDto.LinkDto(WebMvcLinkBuilder.linkTo(method, userId, paymentId).toString(), "POST");
```
Comment (YasharRahvar) [inline api/src/main/java/uk/gov/hmcts/payment/api/dto/mapper/CreditAccountDtoMapper.java:219]: Codecov is complaining this line is not covered. Can you take a look?

File: api/src/main/java/uk/gov/hmcts/payment/api/v1/controllers/PaymentDtoFactory.java [hunk 2]
```diff
@@ -32,7 +32,7 @@ public PaymentOldDto toDto(PaymentOld paymentOld) {
     @SneakyThrows(NoSuchMethodException.class)
     private PaymentOldDto.LinkDto cancellationLink(String userId, Integer paymentId) {
         Method method = PaymentOldController.class.getMethod("cancel", String.class, Integer.class);
-        return new PaymentOldDto.LinkDto(ControllerLinkBuilder.linkTo(method, userId, paymentId).toString(), "POST");
+        return new PaymentOldDto.LinkDto(WebMvcLinkBuilder.linkTo(method, userId, paymentId).toString(), "POST");
```
Comment (YasharRahvar) [inline api/src/main/java/uk/gov/hmcts/payment/api/v1/controllers/PaymentDtoFactory.java:35]: Codecov is complaining this line is not covered. Can you take a look?

**Review by AnilDhawaleGitHub — APPROVED** [review]

**Review by amitchidrewar1301 — APPROVED** [review]

File: api/src/main/java/uk/gov/hmcts/payment/api/dto/mapper/CreditAccountDtoMapper.java (hunk 1 above)
Comment (aravindan89) [reply api/src/main/java/uk/gov/hmcts/payment/api/dto/mapper/CreditAccountDtoMapper.java:219]: Hi @YasharRahvar - This is again a same issue that we faced during apportionment. We are good with the overall coverage and it is not going down.

File: api/src/main/java/uk/gov/hmcts/payment/api/v1/controllers/PaymentDtoFactory.java (hunk 2 above)
Comment (aravindan89) [reply api/src/main/java/uk/gov/hmcts/payment/api/v1/controllers/PaymentDtoFactory.java:35]: Hi @YasharRahvar - This is again a same issue that we faced during apportionment. We are good with the overall coverage and it is not going down.

File: api/src/main/java/uk/gov/hmcts/payment/api/dto/mapper/CreditAccountDtoMapper.java (hunk 1 above)
Comment (YasharRahvar) [reply api/src/main/java/uk/gov/hmcts/payment/api/dto/mapper/CreditAccountDtoMapper.java:219]: ok fine Aravin

**Review by YasharRahvar — APPROVED** [review]

**Review by krishnanuthalapati — APPROVED** [review]

**Pull request merged**